src/m365/teams/commands/meeting/meeting-transcript-list.spec.ts

```diff
@@ -0,0 +1,232 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import appInsights from '../../../../appInsights';
+import auth from '../../../../Auth';
+import { Cli } from '../../../../cli/Cli';
+import { CommandInfo } from '../../../../cli/CommandInfo';
+import { Logger } from '../../../../cli/Logger';
+import Command, { CommandError } from '../../../../Command';
+import request from '../../../../request';
+import { accessToken } from '../../../../utils/accessToken';
+import { pid } from '../../../../utils/pid';
+import { sinonUtil } from '../../../../utils/sinonUtil';
+import commands from '../../commands';
+import * as userGetCommand from '../../../aad/commands/user/user-get';
+const command: Command = require('./meeting-transcript-list');
+
+describe(commands.MEETING_TRANSCRIPT_LIST, () => {
+  const userId = '68be84bf-a585-4776-80b3-30aa5207aa21';
+  const userName = 'user@tenant.com';
+  const meetingId = 'MSo5MWZmMmUxNy04NGRlLTQ1NWEtODgxNS01MmIyMTY4M2Y2NGUqMCoqMTk6bWVldGluZ19ZMlEzTlRRMFpEWXRaamMzWkMwMFlUVmhMVGt4TTJJdFpURmtNMkUwTUdGak1qVmpAdGhyZWFkLnYy';
+  const meetingTranscriptsResponse =
+    [
+      {
+        "id": "MSMjMCMjZDAwYWU3NjUtNmM2Yi00NjQxLTgwMWQtMTkzMmFmMjEzNzdh",
+        "createdDateTime": "2021-09-17T06:09:24.8968037Z"
+      },
+      {
+        "id": "MSMjMCMjMzAxNjNhYTctNWRmZi00MjM3LTg5MGQtNWJhYWZjZTZhNWYw",
+        "createdDateTime": "2021-09-16T18:58:58.6760692Z"
+      },
+      {
+        "id": "MSMjMCMjNzU3ODc2ZDYtOTcwMi00MDhkLWFkNDItOTE2ZDNmZjkwZGY4",
+        "createdDateTime": "2021-09-16T18:56:00.9038309Z"
+      }
+    ];
+
+  let log: string[];
+  let logger: Logger;
+  let loggerLogSpy: sinon.SinonSpy;
+  let commandInfo: CommandInfo;
+
+  before(() => {
+    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
+    sinon.stub(appInsights, 'trackEvent').callsFake(() => { });
```

Comment:
Hi, just popping in to notify you that we should update this line to:

```
sinon.stub(telemetry, 'trackEvent').callsFake(() => { });
```

to align it with our latest change: #4215